### PR TITLE
Add efficiencies to UI.

### PR DESCRIPTION
**Pull request by stephen-smith**
My feature request here: http://www.reddit.com/r/feedthebeast/comments/2r9vwf/bigreactor_simulator/cnedway

I haven't been able to test this code, yet.  Please let me know the best way to verify my changes.

### CODE REVIEW

**Comment by sidoh** [comment]
Beautiful - this seems like a good way to go about it.. As you've seen, there isn't much of a way to test anything in the UI. I was just pasting in a JSON response from the API to simulate production. Not sure of a better way, or if it's worth it for something of this scale, but definitely open to suggestions.

File: assets/scripts/reactor_simulator.js [hunk 1]
```diff
@@ -165,8 +165,28 @@
     return layout;
   };
 
+  /* Adds UI-calculated fields to simulator response. */
+  var augmentResponse = function(response) {
+    var output = response['output']
+    var fuelUse = response['fuelConsumption'];
+    var fuelEff = output / fuel;
```
Comment (sidoh) [inline assets/scripts/reactor_simulator.js:172]: Should `fuel` be `fuelUse`?

**Comment by sidoh** [comment]
The code looks great, but the text labels for the new fields [sorta break the flow](http://i.imgur.com/NAftY9U.png). I realize there isn't an obvious choice for icons, but it'd be great if we could find something that works. I don't think they'd need to be perfectly fitting -- there should probably already be a legend anyway.

File: assets/scripts/reactor_simulator.js (hunk 1 above)
Comment (stephen-smith) [inline assets/scripts/reactor_simulator.js:172]: Yes.  A little incomplete refactoring, there.

**Comment by stephen-smith** [comment]
I can probably throw together some poor icons next time I work on this.  If you want to merge before then, I think adding a <br/> after the leader could work until icons are available.

**Comment by sidoh** [comment]
Actually, I think putting some space between it and the raw simulator results and adding a small header labeled "Efficiency" would work. The units would make it clear what each line refers to. Then the labels wouldn't be necessary.

**Comment by sidoh** [comment]
Gonna merge it do what I said in the last comment. Thanks a bunch of the PR!

**Comment by stephen-smith** [comment]
Thanks!

**Pull request merged**